source/include/mqtt_agent.h

```diff
@@ -268,6 +371,23 @@ MQTTStatus_t MQTTAgent_CommandLoop( MQTTAgentContext_t * pMqttAgentContext );
  *
  * @return #MQTTSuccess if it succeeds in resending publishes, else an
  * appropriate error code from `MQTT_Publish()`
+ *
+ * <b>Example</b>
+ * @code{c}
+ *
+ * // Variables used in this example.
+ * MQTTStatus_t status;
+ * MQTTAgentContext_t mqttAgentContext;
+ * // Value obtained from server through #MQTT_Connect call.
+ * bool sessionPresent;
+ *
+ * status = MQTTAgent_ResumeSession( &mqttAgentContext, sessionPresent );
```

Comment:
Can you add an `MQTT_Connect` call right before this? You don't need to go through the parameters for it, just the function call

Reply:
Good suggestion. Added the call.